src/components/Footer/index.tsx

```diff
@@ -19,30 +19,30 @@ const Link = styled.a`
   text-decoration: none;
 `;
 
-const LanguageContainer = styled.div`
-  display: flex;
-  justify-content: center;
-`;
+// const LanguageContainer = styled.div`
```

Comment:
eslint suggest unused var which is correct because component is commented out for some reason https://github.com/Cryptonomic/Arronax/blob/6797b2905d0bbe20cdb13f860757ebaaee81c321/src/components/Footer/index.tsx#L54

Reply:
I checked now and looks like doesn't work